code-challenges.test.js

```diff
@@ -25,11 +25,56 @@ var people = [
 ]
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is a president of the galaxy.", "Arthur Dent is a radio employee."]
 
+describe("People function", () => {
+  it("Takes in an array of objects and returns an array with a sentence about each person with their names capitalized", () => {
+    const output = ["Ford Prefect is a hitchhiker.",
+      "Zaphod Beeblebrox is a president of the galaxy.",
+      "Arthur Dent is a radio employee."]
+
+    // first test my helper method to capitalize the name
+    expect(capitalizeName(people[0].name)).toEqual("Ford Prefect")
+    
+    // now test people list
+    expect(peopleList(people)).toEqual(output)
+  })
+})
+
+/*
+  function: peopleList
+  param: arr of objects with names and occupations
+  create an array for peopleList - an array of sentences about each person 
+  for each object of the array
+  call a helper function to get the name capitalized
+    split the name by " " to get an array
+    map and toUpperCase() on the first letter
+    join(" ")
+    return a string of the name capitalized
+  use string interpolation to then PUSH onto the peopleList
+  "${person.nameCapitalized} is a ${person.occupation}."
+  return peopleList
+*/
 
 
 // b) Create the function that makes the test pass.
 
-
+// Takes an array of objects, where each object represents a person
+// with key-values for names and occupations,
+// and returns an array of sentences about each person.
+const peopleList = (people) => {
+  let sentences = []
+  people.forEach(person => {
+    let sentence = `${capitalizeName(person.name)} is a ${person.occupation}.`
+    sentences.push(sentence)
+  })
+  return sentences
+}
+
+// Helper method to capitalize the name of a person
+const capitalizeName = (nameLowerCase) => {
+  return nameLowerCase.split(" ")
+                      .map(name => name[0].toUpperCase() + name.substring(1))
+                      .join(" ")
+}
```

Comment:
I think this is smart, breaking the solution into two different utilities. Essentially having one function nested inside of the other. Alright alright, good stuff.